<a href="https://colab.research.google.com/github/sriram333/World-Bank-Global-Education-Analysis/blob/main/Venkata_Sai_Sri_Ram_Chunduri_World_Bank_Global_Education_Analysis_Capstone_Project.ipynb" target="_parent"><img src="https://colab.research.google.com/assets/colab-badge.svg" alt="Open In Colab"/></a>

## <b> The World Bank EdStats All Indicator Query holds over 4,000 internationally comparable indicators that describe education access, progression, completion, literacy, teachers, population, and expenditures.The indicators cover the education cycle from pre-primary to vocational and tertiary education and also holds learning outcome data from international and regional learning assessments (e.g. PISA, TIMSS, PIRLS), equity data from household surveys, and projection/attainment data. </b>

## <b> Explore and analyze the data to identify variation of indicators across the globe, which countries are more alike and different. Feel free to add more extensive analyses and details.</b>

In [1]:
from google.colab import drive
drive.mount('/content/drive')

Mounted at /content/drive


In [3]:
# Import Libraries / Packages 
import pandas as pd
import numpy as np

## **Load the Datasets of World Bank Global Education Statistics**

**EdStatsData.csv**

In [5]:
working_directory = '/content/drive/MyDrive/AlmaBetter/Cohort Nilgiri/Module 1/Capstone project/'
df_stats_data = pd.read_csv(working_directory + 'EdStatsData.csv')

**EdStatsCountry-Series.csv**

In [6]:
df_country_series = pd.read_csv(working_directory + 'EdStatsCountry-Series.csv')

**EdStatsCountry.csv**

In [74]:
df_country = pd.read_csv(working_directory + 'EdStatsCountry.csv')

**EdStatsFootNote.csv**

In [8]:
df_footnote = pd.read_csv(working_directory + 'EdStatsFootNote.csv')

**EdStatsSeries.csv**

In [101]:
df_stats_series = pd.read_csv(working_directory+'EdStatsSeries.csv')

### **Basic Analysis of EdStatsData.csv**

In [10]:
df_stats_data.head()

Country Name Country Code  ... 2100 Unnamed: 69
0   Arab World          ARB  ...  NaN         NaN
1   Arab World          ARB  ...  NaN         NaN
2   Arab World          ARB  ...  NaN         NaN
3   Arab World          ARB  ...  NaN         NaN
4   Arab World          ARB  ...  NaN         NaN

[5 rows x 70 columns]

In [11]:
df_stats_data.columns

Index(['Country Name', 'Country Code', 'Indicator Name', 'Indicator Code',
       '1970', '1971', '1972', '1973', '1974', '1975', '1976', '1977', '1978',
       '1979', '1980', '1981', '1982', '1983', '1984', '1985', '1986', '1987',
       '1988', '1989', '1990', '1991', '1992', '1993', '1994', '1995', '1996',
       '1997', '1998', '1999', '2000', '2001', '2002', '2003', '2004', '2005',
       '2006', '2007', '2008', '2009', '2010', '2011', '2012', '2013', '2014',
       '2015', '2016', '2017', '2020', '2025', '2030', '2035', '2040', '2045',
       '2050', '2055', '2060', '2065', '2070', '2075', '2080', '2085', '2090',
       '2095', '2100', 'Unnamed: 69'],
      dtype='object')

In [13]:
df_stats_data.shape

(886930, 70)

In [14]:
df_stats_data.info()

<class 'pandas.core.frame.DataFrame'>
RangeIndex: 886930 entries, 0 to 886929
Data columns (total 70 columns):
 #   Column          Non-Null Count   Dtype  
---  ------          --------------   -----  
 0   Country Name    886930 non-null  object 
 1   Country Code    886930 non-null  object 
 2   Indicator Name  886930 non-null  object 
 3   Indicator Code  886930 non-null  object 
 4   1970            72288 non-null   float64
 5   1971            35537 non-null   float64
 6   1972            35619 non-null   float64
 7   1973            35545 non-null   float64
 8   1974            35730 non-null   float64
 9   1975            87306 non-null   float64
 10  1976            37483 non-null   float64
 11  1977            37574 non-null   float64
 12  1978            37576 non-null   float64
 13  1979            36809 non-null   float64
 14  1980            89122 non-null   float64
 15  1981            38777 non-null   float64
 16  1982            37511 non-null   float64
 17  1983      

In [17]:
df_stats_data['Country Name'].unique()

array(['Arab World', 'East Asia & Pacific',
       'East Asia & Pacific (excluding high income)', 'Euro area',
       'Europe & Central Asia',
       'Europe & Central Asia (excluding high income)', 'European Union',
       'Heavily indebted poor countries (HIPC)', 'High income',
       'Latin America & Caribbean',
       'Latin America & Caribbean (excluding high income)',
       'Least developed countries: UN classification',
       'Low & middle income', 'Low income', 'Lower middle income',
       'Middle East & North Africa',
       'Middle East & North Africa (excluding high income)',
       'Middle income', 'North America', 'OECD members', 'South Asia',
       'Sub-Saharan Africa', 'Sub-Saharan Africa (excluding high income)',
       'Upper middle income', 'World', 'Afghanistan', 'Albania',
       'Algeria', 'American Samoa', 'Andorra', 'Angola',
       'Antigua and Barbuda', 'Argentina', 'Armenia', 'Aruba',
       'Australia', 'Austria', 'Azerbaijan', 'Bahamas, The', 'Bahrain',


In [21]:
df_stats_data['Country Name'].unique().shape

(242,)

In [19]:
df_stats_data[df_stats_data['Country Name'] == 'High income'].head()

Country Name Country Code  ... 2100 Unnamed: 69
29320  High income          HIC  ...  NaN         NaN
29321  High income          HIC  ...  NaN         NaN
29322  High income          HIC  ...  NaN         NaN
29323  High income          HIC  ...  NaN         NaN
29324  High income          HIC  ...  NaN         NaN

[5 rows x 70 columns]

In [135]:
df_stats_data.loc[~df_stats_data['2017'].isna(),'Indicator Name'].unique()

array(['SABER: (Education Management Information Systems) Policy Goal 1 Lever 1: Legal Framework',
       'SABER: (Education Management Information Systems) Policy Goal 1 Lever 2: Organizational Structure',
       'SABER: (Education Management Information Systems) Policy Goal 1 Lever 3: Human Resources',
       'SABER: (Education Management Information Systems) Policy Goal 1 Lever 4: Infrastructural capacity',
       'SABER: (Education Management Information Systems) Policy Goal 1 Lever 5: Budget',
       'SABER: (Education Management Information Systems) Policy Goal 1 Lever 6: Data-driven Culture',
       'SABER: (Education Management Information Systems) Policy Goal 1: Enabling Environment',
       'SABER: (Education Management Information Systems) Policy Goal 2 Lever 1: Data Architecture',
       'SABER: (Education Management Information Systems) Policy Goal 2 Lever 2: Data Coverage',
       'SABER: (Education Management Information Systems) Policy Goal 2 Lever 3: Data Analytics',
 

In [20]:
df_stats_data['Indicator Name'].unique().shape

(3665,)

In [17]:
df_stats_data.shape

(886930, 70)

**Considering the data from 2000 to 2020 and the rows which have atleast an indicator value. Not considering the rows which have all NaNs and no value.**

In [125]:
cols_required = np.concatenate((np.array(df_stats_data.columns[:4]),np.array(df_stats_data.loc[:,"2000":"2016"].columns)))

In [126]:
cols_required

array(['Country Name', 'Country Code', 'Indicator Name', 'Indicator Code',
       '2000', '2001', '2002', '2003', '2004', '2005', '2006', '2007',
       '2008', '2009', '2010', '2011', '2012', '2013', '2014', '2015',
       '2016'], dtype=object)

In [127]:
new_stats_data = df_stats_data.loc[~df_stats_data.loc[:,"2000":"2016"].isna().all(axis=1),cols_required]

In [128]:
new_stats_data.head()

Country Name Country Code  ... 2015 2016
4    Arab World          ARB  ...  NaN  NaN
5    Arab World          ARB  ...  NaN  NaN
6    Arab World          ARB  ...  NaN  NaN
7    Arab World          ARB  ...  NaN  NaN
12   Arab World          ARB  ...  NaN  NaN

[5 rows x 21 columns]

In [131]:
new_stats_data['Indicator Name'].unique().shape

(3636,)

In [129]:
new_stats_data.info()

<class 'pandas.core.frame.DataFrame'>
Int64Index: 348685 entries, 4 to 886929
Data columns (total 21 columns):
 #   Column          Non-Null Count   Dtype  
---  ------          --------------   -----  
 0   Country Name    348685 non-null  object 
 1   Country Code    348685 non-null  object 
 2   Indicator Name  348685 non-null  object 
 3   Indicator Code  348685 non-null  object 
 4   2000            176676 non-null  float64
 5   2001            123509 non-null  float64
 6   2002            124205 non-null  float64
 7   2003            130363 non-null  float64
 8   2004            128814 non-null  float64
 9   2005            184108 non-null  float64
 10  2006            140312 non-null  float64
 11  2007            137272 non-null  float64
 12  2008            134387 non-null  float64
 13  2009            142108 non-null  float64
 14  2010            242442 non-null  float64
 15  2011            146012 non-null  float64
 16  2012            147264 non-null  float64
 17  2013      

In [130]:
new_stats_data.describe()

2000          2001  ...          2015          2016
count  1.766760e+05  1.235090e+05  ...  1.310580e+05  1.646000e+04
mean   9.423384e+09  1.384867e+10  ...  2.353720e+10  1.934411e+11
std    4.442374e+11  5.449242e+11  ...  9.732465e+11  2.839188e+12
min   -6.759300e+04 -6.973700e+04  ... -2.467847e+00 -1.797539e+00
25%    5.699035e+00  1.599897e+01  ...  4.100000e-01  1.826761e+01
50%    5.078717e+01  8.300000e+02  ...  5.235000e+01  6.264000e+03
75%    3.343950e+04  1.746540e+05  ...  6.153575e+04  5.939590e+05
max    6.327293e+13  6.477834e+13  ...  1.156198e+14  1.206030e+14

[8 rows x 17 columns]

###**Basic analysis of EdStatsCountry-Series.csv**

In [42]:
df_country_series.head()

CountryCode  ... Unnamed: 3
0         ABW  ...        NaN
1         ABW  ...        NaN
2         AFG  ...        NaN
3         AFG  ...        NaN
4         AFG  ...        NaN

[5 rows x 4 columns]

In [44]:
df_country_series.shape

(613, 4)

In [45]:
df_country_series.info()

<class 'pandas.core.frame.DataFrame'>
RangeIndex: 613 entries, 0 to 612
Data columns (total 4 columns):
 #   Column       Non-Null Count  Dtype  
---  ------       --------------  -----  
 0   CountryCode  613 non-null    object 
 1   SeriesCode   613 non-null    object 
 2   DESCRIPTION  613 non-null    object 
 3   Unnamed: 3   0 non-null      float64
dtypes: float64(1), object(3)
memory usage: 19.3+ KB


**Dropping the Unnamed: 3 column from df_country_series dataframe as it has all Nan values**

In [51]:
df_country_series = df_country_series.drop('Unnamed: 3',axis=1)

In [52]:
df_country_series.head()

CountryCode  ...                                        DESCRIPTION
0         ABW  ...  Data sources : United Nations World Population...
1         ABW  ...  Data sources: United Nations World Population ...
2         AFG  ...  Data sources: United Nations World Population ...
3         AFG  ...                 Estimates are based on regression.
4         AFG  ...  Data sources : United Nations World Population...

[5 rows x 3 columns]

In [56]:
df_country_series['CountryCode'].unique()

array(['ABW', 'AFG', 'AGO', 'ALB', 'AND', 'ARE', 'ARG', 'ARM', 'ASM',
       'ATG', 'AUS', 'AUT', 'AZE', 'BDI', 'BEL', 'BEN', 'BFA', 'BGD',
       'BGR', 'BHR', 'BHS', 'BIH', 'BLR', 'BLZ', 'BMU', 'BOL', 'BRA',
       'BRB', 'BRN', 'BTN', 'BWA', 'CAF', 'CAN', 'CHE', 'CHI', 'CHL',
       'CHN', 'CIV', 'CMR', 'COD', 'COG', 'COL', 'COM', 'CPV', 'CRI',
       'CUB', 'CYM', 'CYP', 'CZE', 'DEU', 'DJI', 'DMA', 'DNK', 'DOM',
       'DZA', 'ECU', 'EGY', 'ERI', 'ESP', 'EST', 'ETH', 'FIN', 'FJI',
       'FRA', 'FRO', 'FSM', 'GAB', 'GBR', 'GEO', 'GHA', 'GIB', 'GIN',
       'GMB', 'GNB', 'GNQ', 'GRC', 'GRD', 'GRL', 'GTM', 'GUM', 'GUY',
       'HKG', 'HND', 'HRV', 'HTI', 'HUN', 'IDN', 'IMN', 'IND', 'IRL',
       'IRN', 'IRQ', 'ISL', 'ISR', 'ITA', 'JAM', 'JOR', 'JPN', 'KAZ',
       'KEN', 'KGZ', 'KHM', 'KIR', 'KNA', 'KOR', 'KWT', 'LAO', 'LBN',
       'LBR', 'LBY', 'LCA', 'LIE', 'LKA', 'LSO', 'LTU', 'LUX', 'LVA',
       'MAC', 'MAR', 'MCO', 'MDA', 'MDG', 'MDV', 'MEX', 'MHL', 'MKD',
       'MLI', 'MLT',

In [55]:
df_country_series['CountryCode'].unique().shape

(211,)

In [58]:
df_country_series['SeriesCode'].unique()

array(['SP.POP.TOTL', 'SP.POP.GROW', 'NY.GDP.PCAP.PP.CD',
       'NY.GDP.MKTP.PP.KD', 'NY.GNP.MKTP.PP.CD', 'NY.GDP.MKTP.PP.CD',
       'NY.GDP.PCAP.PP.KD', 'NY.GNP.PCAP.PP.CD', 'SP.POP.1564.TO.ZS',
       'SP.POP.TOTL.MA.ZS', 'SP.POP.TOTL.FE.ZS', 'SP.POP.0014.TO.ZS',
       'NY.GNP.PCAP.CD', 'NY.GDP.PCAP.CD', 'NY.GDP.PCAP.KD',
       'SP.POP.1564.MA.IN', 'SP.POP.0014.TO', 'SP.POP.1564.TO',
       'SP.POP.1564.FE.IN', 'SP.POP.0014.MA.IN', 'SP.POP.0014.FE.IN'],
      dtype=object)

In [60]:
df_country_series['DESCRIPTION'].unique()

array(['Data sources : United Nations World Population Prospects',
       'Data sources: United Nations World Population Prospects',
       'Estimates are based on regression.',
       'Data sources : Institute of Statistics, Eurostat',
       'Data sources : Australian Bureau of Statistics',
       'Including Other Territories comprising Jervis Bay Territory, Christmas Island and the Cocos (Keeling) Islands. ',
       'Data sources: Australian Bureau of Statistics',
       'Data sources : Eurostat',
       'Data sources : Eurostat, State Statistical Committee, United Nations World Population Prospects',
       'Data sources: Eurostat, State Statistical Committee, United Nations World Population Prospects',
       'Data sources : National Statistical Committee of the Republic of Belarus',
       'Data sources: National Statistical Committee of the Republic of Belarus',
       'Data sources : Bermuda Government Department of Statistics',
       'Data sources: Bermuda Government Departme

### **Basic Analysis of EdStatsCountry.csv**

In [75]:
df_country.head()

Country Code   Short Name  ... Latest water withdrawal data Unnamed: 31
0          ABW        Aruba  ...                          NaN         NaN
1          AFG  Afghanistan  ...                         2000         NaN
2          AGO       Angola  ...                         2005         NaN
3          ALB      Albania  ...                         2006         NaN
4          AND      Andorra  ...                          NaN         NaN

[5 rows x 32 columns]

In [112]:
df_country[df_country['Country Code']=='MNA']

Country Code  ... Latest water withdrawal data
148          MNA  ...                          NaN

[1 rows x 28 columns]

In [76]:
df_country.shape

(241, 32)

In [77]:
df_country.columns

Index(['Country Code', 'Short Name', 'Table Name', 'Long Name', '2-alpha code',
       'Currency Unit', 'Special Notes', 'Region', 'Income Group', 'WB-2 code',
       'National accounts base year', 'National accounts reference year',
       'SNA price valuation', 'Lending category', 'Other groups',
       'System of National Accounts', 'Alternative conversion factor',
       'PPP survey year', 'Balance of Payments Manual in use',
       'External debt Reporting status', 'System of trade',
       'Government Accounting concept', 'IMF data dissemination standard',
       'Latest population census', 'Latest household survey',
       'Source of most recent Income and expenditure data',
       'Vital registration complete', 'Latest agricultural census',
       'Latest industrial data', 'Latest trade data',
       'Latest water withdrawal data', 'Unnamed: 31'],
      dtype='object')

In [78]:
df_country.info()

<class 'pandas.core.frame.DataFrame'>
RangeIndex: 241 entries, 0 to 240
Data columns (total 32 columns):
 #   Column                                             Non-Null Count  Dtype  
---  ------                                             --------------  -----  
 0   Country Code                                       241 non-null    object 
 1   Short Name                                         241 non-null    object 
 2   Table Name                                         241 non-null    object 
 3   Long Name                                          241 non-null    object 
 4   2-alpha code                                       238 non-null    object 
 5   Currency Unit                                      215 non-null    object 
 6   Special Notes                                      145 non-null    object 
 7   Region                                             214 non-null    object 
 8   Income Group                                       214 non-null    object 
 9   WB-2 code 

In [79]:
df_country['Country Code'].unique().shape

(241,)

In [80]:
df_country['Short Name'].unique().shape

(241,)

In [81]:
df_country['Income Group'].unique()

array(['High income: nonOECD', 'Low income', 'Upper middle income', nan,
       'Lower middle income', 'High income: OECD'], dtype=object)

In [82]:
df_country['Region'].unique()

array(['Latin America & Caribbean', 'South Asia', 'Sub-Saharan Africa',
       'Europe & Central Asia', nan, 'Middle East & North Africa',
       'East Asia & Pacific', 'North America'], dtype=object)

**Dropping the Unnamed: 31 column as it has all Nan values and columns like National accounts reference year, Alternative conversion factor and Other groups which have maximum Nans and less values**

In [83]:
df_country = df_country.drop(['Unnamed: 31','National accounts reference year','Alternative conversion factor','Other groups'],axis=1)

In [84]:
df_country.shape

(241, 28)

In [85]:
df_country[df_country.loc[:,'2-alpha code':'Latest water withdrawal data'].isna().all(axis=1)]

Empty DataFrame
Columns: [Country Code, Short Name, Table Name, Long Name, 2-alpha code, Currency Unit, Special Notes, Region, Income Group, WB-2 code, National accounts base year, SNA price valuation, Lending category, System of National Accounts, PPP survey year, Balance of Payments Manual in use, External debt Reporting status, System of trade, Government Accounting concept, IMF data dissemination standard, Latest population census, Latest household survey, Source of most recent Income and expenditure data, Vital registration complete, Latest agricultural census, Latest industrial data, Latest trade data, Latest water withdrawal data]
Index: []

### **Basic Analysis of EdStatsFootNote.csv dataset**

In [86]:
df_footnote.head()

CountryCode      SeriesCode    Year          DESCRIPTION  Unnamed: 4
0         ABW  SE.PRE.ENRL.FE  YR2001  Country estimation.         NaN
1         ABW  SE.TER.TCHR.FE  YR2005  Country estimation.         NaN
2         ABW  SE.PRE.TCHR.FE  YR2000  Country estimation.         NaN
3         ABW  SE.SEC.ENRL.GC  YR2004  Country estimation.         NaN
4         ABW     SE.PRE.TCHR  YR2006  Country estimation.         NaN

In [87]:
df_footnote.columns

Index(['CountryCode', 'SeriesCode', 'Year', 'DESCRIPTION', 'Unnamed: 4'], dtype='object')

In [88]:
df_footnote.info()

<class 'pandas.core.frame.DataFrame'>
RangeIndex: 643638 entries, 0 to 643637
Data columns (total 5 columns):
 #   Column       Non-Null Count   Dtype  
---  ------       --------------   -----  
 0   CountryCode  643638 non-null  object 
 1   SeriesCode   643638 non-null  object 
 2   Year         643638 non-null  object 
 3   DESCRIPTION  643638 non-null  object 
 4   Unnamed: 4   0 non-null       float64
dtypes: float64(1), object(4)
memory usage: 24.6+ MB


**Dropping Unnamed: 4 column which has all Nan values**

In [89]:
df_footnote = df_footnote.drop('Unnamed: 4',axis=1)

In [90]:
df_footnote.shape

(643638, 4)

In [91]:
df_footnote['CountryCode'].unique().shape

(239,)

In [92]:
df_footnote['SeriesCode'].unique().shape

(1558,)

In [93]:
df_footnote['DESCRIPTION'].unique().shape

(9102,)

### **Basic Analysis of EdStatsSeries.csv** 

In [94]:
df_stats_series.head()

Series Code       Topic  ... License Type Unnamed: 20
0  BAR.NOED.1519.FE.ZS  Attainment  ...          NaN         NaN
1     BAR.NOED.1519.ZS  Attainment  ...          NaN         NaN
2  BAR.NOED.15UP.FE.ZS  Attainment  ...          NaN         NaN
3     BAR.NOED.15UP.ZS  Attainment  ...          NaN         NaN
4  BAR.NOED.2024.FE.ZS  Attainment  ...          NaN         NaN

[5 rows x 21 columns]

In [95]:
df_stats_series.info()

<class 'pandas.core.frame.DataFrame'>
RangeIndex: 3665 entries, 0 to 3664
Data columns (total 21 columns):
 #   Column                               Non-Null Count  Dtype  
---  ------                               --------------  -----  
 0   Series Code                          3665 non-null   object 
 1   Topic                                3665 non-null   object 
 2   Indicator Name                       3665 non-null   object 
 3   Short definition                     2156 non-null   object 
 4   Long definition                      3665 non-null   object 
 5   Unit of measure                      0 non-null      float64
 6   Periodicity                          99 non-null     object 
 7   Base Period                          314 non-null    object 
 8   Other notes                          552 non-null    object 
 9   Aggregation method                   47 non-null     object 
 10  Limitations and exceptions           14 non-null     object 
 11  Notes from original source    

**Dropping columns like Unnamed: 20, Other web links, Related indicators, License Type, Statistical concept and methodology, Development relevance, General comments, Notes from original source, Limitations and exceptions, Unit of measure,Aggregation method** 




In [102]:
df_stats_series = df_stats_series.drop(['Unnamed: 20', 'Other web links', 'Related indicators', 'License Type', 'Statistical concept and methodology', 'Development relevance', 'General comments', 'Notes from original source', 'Limitations and exceptions', 'Unit of measure','Aggregation method'],axis=1)

In [103]:
df_stats_series.info()

<class 'pandas.core.frame.DataFrame'>
RangeIndex: 3665 entries, 0 to 3664
Data columns (total 10 columns):
 #   Column                Non-Null Count  Dtype 
---  ------                --------------  ----- 
 0   Series Code           3665 non-null   object
 1   Topic                 3665 non-null   object
 2   Indicator Name        3665 non-null   object
 3   Short definition      2156 non-null   object
 4   Long definition       3665 non-null   object
 5   Periodicity           99 non-null     object
 6   Base Period           314 non-null    object
 7   Other notes           552 non-null    object
 8   Source                3665 non-null   object
 9   Related source links  215 non-null    object
dtypes: object(10)
memory usage: 286.5+ KB


In [104]:
df_stats_series['Series Code'].unique().shape


(3665,)

In [107]:
df_stats_series['Topic'].unique()

array(['Attainment', 'Education Equality',
       'Infrastructure: Communications', 'Learning Outcomes',
       'Economic Policy & Debt: National accounts: US$ at current prices: Aggregate indicators',
       'Economic Policy & Debt: National accounts: US$ at constant 2010 prices: Aggregate indicators',
       'Economic Policy & Debt: Purchasing power parity',
       'Economic Policy & Debt: National accounts: Atlas GNI & GNI per capita',
       'Teachers', 'Education Management Information Systems (SABER)',
       'Early Child Development (SABER)',
       'Engaging the Private Sector (SABER)',
       'School Health and School Feeding (SABER)',
       'School Autonomy and Accountability (SABER)',
       'School Finance (SABER)', 'Student Assessment (SABER)',
       'Teachers (SABER)', 'Tertiary Education (SABER)',
       'Workforce Development (SABER)', 'Literacy', 'Background',
       'Primary', 'Secondary', 'Tertiary', 'Early Childhood Education',
       'Pre-Primary', 'Expenditures'

In [106]:
df_stats_series['Indicator Name'].unique().shape

(3665,)

**Findings from the Dataset :**



*   df_country contains a list of all participating countries (total of 241)

* df_stats_data contains a combination of each country and each indicator (total of 3665 indicators) as well as measurment values for these indicators from 1970 to 2017 and projections from 2020 to 2100.

* df_stats_series contains the list of all indicators, their category group and further additional information

* df_country_series contains general information about entire indicators that have been estimated for certain countries

* df_footnote contains informations about the uncertainty and origin of the data series used for the indicators for every single year 



